src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -717,71 +717,13 @@ For example, if the page that was copied from is called `Leads`, then the newly
 }
 ----
 
-[#setting-page-icon]
-== Setting page Icon
+== Configuring Page Icons
 
-Icon can be set to a record list or record page.
+Icon can be set to a list or record page.
 This gives us the ability to further customize a page and easily distinguish it from other pages.
 
-To set a page icon, navigate to its `Edit page` view.
-It can be accessed in the admin area of Corteza under pages in the sidebar.
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-public-view.png",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "annotations": [{
-    "x": 15,
-    "y": 135,
-    "w": 290,
-    "h": 40
-  }]
-}
-----
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-tree-case-management.png",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "annotations": [{
-    "x": 1520,
-    "y": 225,
-    "w": 70,
-    "h": 25
-  }]
-}
-----
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-edit.png",
-    "alias": "page-edit-base",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {},
-  "annotations": [{
-    "x": 590,
-    "y": 440,
-    "w": 95,
-    "h": 60
-  }]
-}
-----
-
-Click on the icon to set page icon.
+To set the page icon, navigate to the menu:admin panel[pages], and edit the page you wish to set the icon for.
```

Comment:
Perhaps we should mention that these steps apply after you're inside of your namespace already.

It's sort of intuitive given that the above content already talks about it so I'm fine with either.